Package/Repositories/HttpPackageRepository.cs

```diff
@@ -695,8 +548,8 @@ public JObject Query(string query)
         /// </summary>
         /// <param name="query">A GraphQL query string</param>
         /// <returns>A JSON string containing the GraphQL response</returns>
-        public string QueryGraphQL(string query) =>
-            downloadPackagesString($"/3.1/query", query, "application/json", "application/json");
+        public string QueryGraphQL(string query) => downloadPackagesString($"/3.1/query", query, "application/json", "application/json");
```

Comment:
Could we add obsolete attribute and remove this in 9.22? If so, could we also do this for the method above